Sources/PrivacyDashboard/BrokenSiteReporting/PerformanceMetricsSubfeature.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  PerformanceMetricsSubfeature.swift
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import UserScript
+import WebKit
+
+public class PerformanceMetricsSubfeature: Subfeature {
+
+    public var messageOriginPolicy: MessageOriginPolicy = .all
+    public var featureName: String = "performanceMetrics"
+    public var broker: UserScriptMessageBroker?
+
+    public var targetWebview: WKWebView?
```

Comment:
Im not a big fan of exposing this, can't we bind lifecycle of this PerformanceMetrics instance with targetWebview?